tests/DivisionsTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+include_once 'FetchPageTestCase.php';
+
+/**
+ * Provides test methods to ensure pages contain what we want them to in various ways.
+ */
+class DivisionsTest extends FetchPageTestCase
+{
+
+    /**
+     * Connects to the testing database.
+     */
+    public function getConnection()
+    {
+        $dsn = 'mysql:host=' . OPTION_TWFY_DB_HOST . ' ;dbname=' . OPTION_TWFY_DB_NAME;
+        $username = OPTION_TWFY_DB_USER;
+        $password = OPTION_TWFY_DB_PASS;
+        $pdo = new PDO($dsn, $username, $password);
+        return $this->createDefaultDBConnection($pdo, OPTION_TWFY_DB_NAME);
+    }
+
+    public function getDataSet()
+    {
+        return $this->createMySQLXMLDataSet(dirname(__FILE__) . '/_fixtures/divisions.xml');
+    }
+
+    private function fetch_page($vars)
+    {
+        return $this->base_fetch_page('', $vars, 'www/docs/mp', 'index.php', 'REQUEST_URI=mp/divisions.php');
+    }
+
+    private function fetch_division_page() {
+        return $this->fetch_page( array( 'pagetype' => 'divisions', 'pid' => 2, 'policy' => 363, 'url' => '/mp/2/test_current-mp/test_westminster_constituency/divisions' ) );
+    }
+
+
+    public function testVoteDirection() {
+        $page = $this->fetch_division_page();
+        $this->assertContains('Test Current-MP voted Agreed', $page);
+    }
+
+    public function testPolicyDirection() {
+        $page = $this->fetch_division_page();
+        $this->assertContains('voted strongly against introducing <b>foundation', $page);
+    }
+
+    public function testVotedAgainst() {
+        $page = $this->fetch_division_page();
+        $this->assertContains('Test Current-MP voted Do not agree', $page);
```

Comment:
I guess it might be nice to check the Do note agree and Agreed are on the right lines :)